exporter/kafkaexporter/factory.go

```diff
@@ -117,12 +117,6 @@ func createLogsExporter(
 	cfg component.Config,
 ) (exporter.Logs, error) {
 	oCfg := *(cfg.(*Config)) // Clone the config
-	if oCfg.Topic == "" {
-		oCfg.Topic = defaultLogsTopic
-	}
-	if oCfg.Encoding == "otlp_json" {
-		set.Logger.Info("otlp_json is considered experimental and should not be used in a production environment")
```

Comment:
is this worth removing first in its own PR?

Reply:
Sure thing, I'll open one shortly.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39222